Misc/NEWS.d/next/Library/2024-09-16-23-35-08.gh-issue-124130.bKRyYk.rst

```diff
@@ -0,0 +1,4 @@
+The category "\B" in re currently does not match empty strings, which is
+against the general expectation. This patch fixes the problem, affecting
+sre operation "SRE_AT_NON_BOUNDARY", "SRE_AT_LOC_NON_BOUNDARY" and
+"SRE_AT_UNI_NON_BOUNDARY".
```

Comment:
Maybe you can try to shorten NEWS appropriately.

Reply:
This is probably not merged as indicated in discussion of the original issue. Anyhow thanks for pointing out.